### PR TITLE
Fix issue #1875

### PR DESCRIPTION
**Pull request by mprokop**

### CODE REVIEW

File: agent/util.go [hunk 1]
```diff
@@ -59,6 +74,15 @@ func ExecScript(script string) (*exec.Cmd, error) {
 		shell = other
 	}
 	cmd := exec.Command(shell, flag, script)
+
+	if runtime.GOOS == "windows" {
```
Comment (magiconair) [inline agent/util.go]: I think you can reduce this to 

```go
// Windows needs the command line arguments without escaping.
if runtime.GOOS == "windows" {
    cmd.SysProcAttr = &syscall.SysProcAttr{
        CmdLine: strings.Join(cmd.Args, " "),
    }
}
```

**Comment by mprokop** [comment]
This changes are working for me on windows. I can not test this on other platforms.

File: agent/util_windows.go [hunk 2]
```diff
@@ -15,7 +15,8 @@ func ExecScript(script string) (*exec.Cmd, error) {
 	if other := os.Getenv("SHELL"); other != "" {
 		shell = other
 	}
-	cmd := exec.Command(shell, "/C", script)
+	scriptQuoted := "\"" + script + "\"";
```
Comment (magiconair) [inline agent/util_windows.go]: You can use this since `script` is already local

```go
script = "\"" + script + "\""
```

**Comment by magiconair** [comment]
Squashed and merged to master. Thank you!

**Pull request merged**